app/lib/ostatus/activity/creation.rb

```diff
@@ -67,7 +62,7 @@ def process_status
   end
 
   def perform_via_activitypub
-    [find_status(activitypub_uri) || ActivityPub::FetchRemoteStatusService.new.call(activitypub_uri), false]
+    [find_status(activitypub_uri) || ActivityPub::FetchRemoteStatusService.new.call(activitypub_uri, id: false), false]
```

Comment:
Why don't you remove this method entirely?

Note: This method also does deduplication when OS payload handled just after AP payload, but it's not needed now because we use AP id for OS id for new posts...probably.